.github/workflows/tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: setup requirements
         run: |
           python -m pip install --upgrade pip
-          python -m pip install jinja2 markupsafe zc.lockfile
+          python -m pip install jinja2 markupsafe zc.lockfile requests
```

Comment:
Where is requests used?

```console
$ git grep requests
.github/workflows/tests.yml:          python -m pip install jinja2 markupsafe zc.lockfile requests
```

Reply:
![image](https://user-images.githubusercontent.com/9087854/196046045-644573d7-7c1f-4ed2-8382-d200219e947a.png)

This must be why...

Reply:
> Where is requests used?

To issue PURGE "commands" to the CDN.